core/src/crds_value.rs

```diff
@@ -86,13 +87,13 @@ pub struct EpochIncompleteSlots {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
-pub struct SnapshotHash {
+pub struct BankHash {
```

Comment:
Why is this a BankHash?  I thought we are not mixing in our account state into this

Reply:
Sure, I can call it AccountsHash.

Reply:
Actually I used that already. I was just trying to come up with a generic name for the hash, since the hashes are not necessarily just hashes of snapshots.

Reply:
They are the same hash value, computed from the full accounts state. Just the snapshots one is published when a snapshot is made, the AccountsHash one is published on every snapshot interval for now, but it's whenever the hash is calculated.

Reply:
Don’t use BankHash, since it means something :).  How about SnapshotHash?

Reply:
Done